PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml

```diff
@@ -10,8 +10,8 @@
              WindowStartupLocation="CenterScreen"
              Name="Window"
              Title="Picture Slides Lab"
-             Width="1200"
-             Height="700"
+             
+                           
              WindowTitleBrush="#0070C0"
```

Comment:
Only these 4 lines are changed by me. The others are auto changed by the Visual Studio.